eng/packaging.targets

```diff
@@ -125,4 +125,8 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ValidatePackageReadmeExists" Condition="'$(IsShipping)' == 'true' and !Exists('$(PackageReadmeFile)')">
```

Comment:
Why not IsPackable == true?

Reply:
The whole file is already conditioned on that.